docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -488,6 +501,21 @@ and a file system location of ``/var/my_tables/test_table``::
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
 
+The table definition below specifies format ORC, partitioning by columns ``c1`` and ``c2``,
+bloom filter index by columns ``c1`` and ``c2``, fpp is 0.05,
+and a file system location of ``/var/my_tables/test_table``::
+
+    CREATE TABLE test_table (
+        c1 integer,
+        c2 date,
+        c3 double)
+    WITH (
+        format = 'ORC',
+        partitioning = ARRAY['c1', 'c2'],
```

Comment:
Do we need to specify partitioning in this example ? Seems unrelated to bloom filter config

Reply:
Yeah, fixed.